packages/bpk-component-calendar/src/Week.js

```diff
@@ -205,39 +205,41 @@ class Week extends Component {
 
     return (
       <tr className={getClassName('bpk-calendar-grid__week')}>
-        {this.props.dates.map(date => (
-          <DateContainer
-            className={this.props.cellClassName}
-            isEmptyCell={!isSameMonth(date, month) && ignoreOutsideDate}
-            key={date.getDate()}
-            weekendStart={
-              showWeekendSeparator && firstDayOfWeekendIndex === getDay(date)
-            }
-            weekendEnd={
-              showWeekendSeparator && lastDayOfWeekendIndex === getDay(date)
-            }
-          >
-            <DateComponent
-              date={date}
-              modifiers={dateModifiers}
-              aria-label={formatDateFull(date)}
-              onClick={onDateClick}
-              onDateKeyDown={onDateKeyDown}
-              preventKeyboardFocus={preventKeyboardFocus}
-              isKeyboardFocusable={isKeyboardFocusable}
-              isFocused={isSameDay(date, focusedDate)}
-              isSelected={isSameDay(date, selectedDate)}
-              isBlocked={
-                minDate && maxDate
-                  ? !isWithinRange(date, minDate, maxDate)
-                  : false
+        {this.props.dates.map(date => {
+          const isBlocked =
+            minDate && maxDate ? !isWithinRange(date, minDate, maxDate) : false;
+
+          return (
+            <DateContainer
```

Comment:
By default, VoiceOver navigates through each cell of a table. As these are now essentially empty, I think we need to apply `aria-hidden` to each `DateContainer` too, to avoid this:
![Screenshot 2021-03-23 at 08 33 14 (2)](https://user-images.githubusercontent.com/30267516/112123211-f7a59780-8bb8-11eb-8ce1-5610530ccaee.png)

Reply:
Hmm, this is with `control + option + arrow`? 

When using TalkBack it completely skips these boxes.

Reply:
Let me have a play adjusting for desktop 👀

Reply:
Pushed - in my testing this now works. Added to `DateContainer`, so could remove from `DateComponent` I think?

Reply:
Removed from `DateComponent`